resources/digital_systems.md

```diff
@@ -0,0 +1,145 @@
+# Digital Systems
+
+This guide is meant to quickly get OTR Members trained in Digital Systems.
+
+---
+
+<details markdown="1">
+  <summary>Table of Contents</summary>
+
+-
+
+</details>
+
+---
+
+## Intro to Digital Systems
+
+Digital Systems is the backbone of modern technology, encompassing the principles and techniques used to design and analyze systems that process digital information. These systems are integral to a vast array of applications, from simple electronic devices to complex computer networks and advanced communication systems. 
+
+---
+
+## Digital Logic
+
+Digital Logic is the foundation of modern digital systems, involving the principles and techniques used to design and analyze circuits that perform logical operations on binary data. It encompasses the use of logic gates such as AND, OR, NOT, NAND, NOR, XOR, and XNOR, which combine to form more complex circuits. These circuits can perform operations like addition, subtraction, multiplexing, and data storage. Boolean algebra underpins digital logic, providing a mathematical framework to simplify and optimize logical expressions. Additionally, tools like Karnaugh maps help in minimizing logic functions, ensuring efficient circuit design. Digital logic also includes combinational and sequential logic, where combinational logic produces outputs based solely on current inputs, and sequential logic incorporates memory elements to consider past inputs as well.
+
+---
+
+## Truth Tables
+
+Truth tables are essential tools in digital logic, used to represent the output of a logic circuit for all possible input combinations. They provide a clear and systematic way to visualize how a logic circuit responds to different inputs, ensuring accurate design and analysis. Each row of a truth table corresponds to a unique combination of input values, and the columns represent the outputs for those inputs.
+
+To create a truth table, list all possible input combinations, typically in binary form. For example, a truth table for a simple AND gate with two inputs, A and B, would look like this:
+
+| A | B | Output (A AND B) |
+|---|---|------------------|
+| 0 | 0 | 0                |
+| 0 | 1 | 0                |
+| 1 | 0 | 0                |
+| 1 | 1 | 1                |
```

Comment:
Very minor/picky point, for a truth table I would recommend a table with centred values (0/1), see [Formatting content within your table](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-tables#formatting-content-within-your-table).

Reply:
fixed for both tables. added  explicit text algnment.